src/wiki/help-pages/Change-Themes.md

```diff
@@ -0,0 +1,135 @@
+---
+eleventyNavigation:
+  key: Change Themes
+  parent: Help pages
+--- 
+
+# How to change your Themes in Prism Launcher
+
+Themes can help you spice up your launcher appearance
+
+Prism Launcher 6.0 will add support for multiple custom themes at once
+
+***This section will be updated accordingly***
+
+Some themes are only on discord for now, but we will try to transfer them to our repository.
+
+Prism Launcher use Qt stylesheet, to make themes please look at [Qt Documentation](https://doc.qt.io/qt-6/stylesheet-reference.html).
+
+## Themes Downloads
+
+[![Prism Themes Repository](https://img.shields.io/badge/Prism--Themes--Repository-76AE88?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes)
+
+### Themes Currently Available
+
+[![Breeze Dark](https://img.shields.io/badge/Breeze--Dark-blue?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Breeze-Dark.zip)
```

Comment:
Just link to the repo instead of this hardcoded list of themes

Reply:
I'm going to maintain this page so all the badges will be updated when new themes are submitted